articles/serializers.py

```diff
@@ -0,0 +1,28 @@
+import nh3
+from drf_spectacular.utils import extend_schema_field
+from rest_framework import serializers
+
+from articles import models
+
+
+@extend_schema_field(str)
+class SanitizedHtmlField(serializers.Field):
+    @staticmethod
+    def to_representation(value):
+        return value
+
+    def to_internal_value(self, data):
+        return nh3.clean(data)
```

Comment:
See https://github.com/mitodl/hq/discussions/2600 for `nh3` / sanitization discussion.

This is currently too agressive. By default, `nh3.clean(data)` strips classes and data-attributes, which we use for things like embedly cards.

I'd like to configure nh3 more separately from this PR: need experiment more with CKEditor, too, to see exactly what tags we need.